src/main/kotlin/cmds/ChestSearchCommand.kt

```diff
@@ -0,0 +1,85 @@
+package cmds
+
+import ExactMatchingContainersSearchResult
+import Hound
+import PartialMatchingContainersSearchResult
+import co.aikar.commands.BaseCommand
+import co.aikar.commands.annotation.*
+import data.PartialMatchModel
+import org.bukkit.entity.Player
+
+@CommandAlias("chound|cf")
+class ChestSearchCommand(private val hound: Hound) : BaseCommand() {
+
+
+    @Default
+    @Syntax("<item>")
+    @CommandCompletion("@items")
+    @CommandPermission("hound.search.container")
+    @Description("Highlight item in nearby containers.")
+    fun search(player: Player, model: PartialMatchModel) {
+
+        val radius = hound.searchRadius
+
+        val result = if (model.exact != null && model.fuzzy.isEmpty()) {
+            hound.highlightItemTypeForPlayer(model.exact, player)
+        } else {
+            hound.highlightPartialMatchingItemTypesForPlayer(model.exact, model.fuzzy.toList(), player)
+        }
+
+        val response = when (result) {
+            is ExactMatchingContainersSearchResult -> {
+                exactMatchesResultMessage(model.input, result)
+            }
+            is PartialMatchingContainersSearchResult -> {
+                partialMatchesResultMessage(model.input, result)
+            }
+            null -> {
+                "§4Couldn't find items matching '${model.input}' in a container within $radius ${if (radius == 1) "block" else "blocks"} of you."
+            }
+        }
+
+        player.sendMessage(response)
+    }
+
+
+    private fun String.plural(count: Int, suffix: String) = if (count == 1) this else "$this$suffix"
+
```

Comment:
```suggestion
    private fun String.plural(count: Int, suffix: String) = if (count == 1) this else "$this$suffix"
```

Some inconsistent 2 line breaks. Correct me if this is a Kotlin convention or something.